Automattic/vip-cli
# Adding SQL import to dev environment

## Description

This PR adds the ability to import SQL files containing a WordPress database into the dev environment. 

The added command relies on `wp db import` and relies in the fact that Lando mounts the `$HOME` folder to `/user` in the Docker machine. Thus, this command is essentially an alias to `vip dev-env exec -- wp db import somefile.sql` but with some path handling for that file.

## Steps to Test

1. Check out PR and build the CLI.
2. Create a new environment.
3. Get a SQL that contains a WordPress site.
4. Import the site by running `vip dev-env import path-to-file.sql`
